src/composables/useBcrosNavigate.ts

```diff
@@ -8,9 +8,13 @@ export const useBcrosNavigate = () => {
     // get account id and set in params
     const redirectURL = new URL(url)
     const accountId = account.currentAccount.id
+    const businessId = business.currentBusiness?.identifier
```

Comment:
Seeing as line 51 doesn't use conditional chaining for the identifier, you don't need it either.

I prefer to not use conditional chaining unless it's needed -- and then, you need to use carefully -- because it can mask errors that you should be handling. It's a double-edged sword.